config/boards/mba8mpxl-ras314.wip

```diff
@@ -0,0 +1,9 @@
+# MBa8MPxL-RAS314 with TQMa8MPQL
```

Comment:
Any variant of `TQMa8MPxL` can be used, not just the quad-core one.

Reply:
OK, was changed.